### ISSUE TITLE
Single window + Fullscreen, yuzu forgets fullscreen settings

### ISSUE THREAD

**Issue by Hexaae** [yuzu-emu/yuzu#7428]
BUG:
1. Set menu options **View > Single Window Mode + Fullscreen (F11)**
2. quit yuzu

on next launch yuzu's Fullscreen is always disabled and won't keep setting.
If you disable Single Window Mode instead every game will be launched in fullscreen as expected and setting will remain.

_Gaming laptop ASUS GL703GS, Win 10, i7-8750H (6+6 cores), NVidia 1070 8GB, 32GB RAM, 1920x1080 144Hz G-Sync laptop screen, NVMe + SSD_

**Comment by german77** [comment]
Fullscreen is not persistent since a parameter was added to launch yuzu in fullscreen mode.  Add -f to the shortcut to always start with fullscreen enabled 

`yuzu.exe -f`

**Comment by Hexaae** [comment]
I'd expect the menu option to be permanent on quit & relaunch... :-\

**Comment by delphir-com** [comment]
I agree with @Hexaae , "preserving" the setting is a way more natural...

**Comment by Hexaae** [comment]
Seems solved to me: yuzu now keeps the fullscreen setting on quit and next relaunch.
